src/synopsys-action/utility.ts

```diff
@@ -27,3 +27,13 @@ export async function cleanupTempDir(tempDir: string): Promise<void> {
 export function checkIfGithubHostedAndLinux(): boolean {
   return String(process.env['RUNNER_NAME']).includes('Hosted Agent') && (process.platform === 'linux' || process.platform === 'darwin')
 }
+
+export function formatAndGetErrorMessage(errorMessage: string): string {
+  const appErrorPart = errorMessage.match(".*'([^']*)'.*")?.at(1)
```

Comment:
what is appErrorPart?

Reply:
This is bridge path in the error message '/home/runner/synopsys-bridge/bridge'